### ISSUE TITLE
the loss turns negative

### ISSUE THREAD

**Issue by backkon** [cornellius-gp/gpytorch#1264]
The loss function I use is the negative marginal log likelihood，like this:
`mll = gpytorch.mlls.ExactMarginalLogLikelihood(likelihood, model)
loss = -mll(output, y_train)
`
When training, the loss turns negative at the end, which means the marginal likelihood is greater than 1, it is not consistent with the actual situation.
I don't know where I have been thoughtless, looking forward to your correction.

The training information:
`Iter 1/50 - Loss: 1.913   lengthscale: 0.693   noise: 0.693
Iter 2/50 - Loss: 1.798   lengthscale: 0.744   noise: 0.744
Iter 3/50 - Loss: 1.708   lengthscale: 0.789   noise: 0.795
Iter 4/50 - Loss: 1.636   lengthscale: 0.833   noise: 0.841
Iter 5/50 - Loss: 1.578   lengthscale: 0.878   noise: 0.875
Iter 6/50 - Loss: 1.528   lengthscale: 0.923   noise: 0.895
Iter 7/50 - Loss: 1.483   lengthscale: 0.971   noise: 0.898
Iter 8/50 - Loss: 1.439   lengthscale: 1.021   noise: 0.888
Iter 9/50 - Loss: 1.395   lengthscale: 1.073   noise: 0.866
Iter 10/50 - Loss: 1.353   lengthscale: 1.127   noise: 0.837
Iter 11/50 - Loss: 1.310   lengthscale: 1.185   noise: 0.802
Iter 12/50 - Loss: 1.268   lengthscale: 1.246   noise: 0.763
Iter 13/50 - Loss: 1.227   lengthscale: 1.310   noise: 0.722
Iter 14/50 - Loss: 1.187   lengthscale: 1.378   noise: 0.679
Iter 15/50 - Loss: 1.148   lengthscale: 1.450   noise: 0.636
Iter 16/50 - Loss: 1.111   lengthscale: 1.524   noise: 0.594
Iter 17/50 - Loss: 1.075   lengthscale: 1.602   noise: 0.552
Iter 18/50 - Loss: 1.040   lengthscale: 1.684   noise: 0.511
Iter 19/50 - Loss: 1.007   lengthscale: 1.768   noise: 0.471
Iter 20/50 - Loss: 0.975   lengthscale: 1.856   noise: 0.434
Iter 21/50 - Loss: 0.941   lengthscale: 1.947   noise: 0.398
Iter 22/50 - Loss: 0.907   lengthscale: 2.041   noise: 0.364
Iter 23/50 - Loss: 0.872   lengthscale: 2.138   noise: 0.332
Iter 24/50 - Loss: 0.834   lengthscale: 2.238   noise: 0.303
Iter 25/50 - Loss: 0.792   lengthscale: 2.342   noise: 0.275
Iter 26/50 - Loss: 0.749   lengthscale: 2.449   noise: 0.249
Iter 27/50 - Loss: 0.701   lengthscale: 2.560   noise: 0.226
Iter 28/50 - Loss: 0.649   lengthscale: 2.674   noise: 0.204
Iter 29/50 - Loss: 0.597   lengthscale: 2.790   noise: 0.184
Iter 30/50 - Loss: 0.542   lengthscale: 2.909   noise: 0.165
Iter 31/50 - Loss: 0.487   lengthscale: 3.031   noise: 0.148
Iter 32/50 - Loss: 0.429   lengthscale: 3.154   noise: 0.133
Iter 33/50 - Loss: 0.372   lengthscale: 3.280   noise: 0.119
Iter 34/50 - Loss: 0.316   lengthscale: 3.406   noise: 0.106
Iter 35/50 - Loss: 0.262   lengthscale: 3.534   noise: 0.094
Iter 36/50 - Loss: 0.208   lengthscale: 3.663   noise: 0.084
Iter 37/50 - Loss: 0.151   lengthscale: 3.793   noise: 0.074
Iter 38/50 - Loss: 0.095   lengthscale: 3.924   noise: 0.066
Iter 39/50 - Loss: 0.042   lengthscale: 4.055   noise: 0.058
Iter 40/50 - Loss: -0.014   lengthscale: 4.187   noise: 0.051
Iter 41/50 - Loss: -0.070   lengthscale: 4.319   noise: 0.045
Iter 42/50 - Loss: -0.130   lengthscale: 4.451   noise: 0.040
Iter 43/50 - Loss: -0.187   lengthscale: 4.584   noise: 0.035
Iter 44/50 - Loss: -0.243   lengthscale: 4.716   noise: 0.031
Iter 45/50 - Loss: -0.307   lengthscale: 4.849   noise: 0.027
Iter 46/50 - Loss: -0.368   lengthscale: 4.981   noise: 0.024
Iter 47/50 - Loss: -0.430   lengthscale: 5.112   noise: 0.021
Iter 48/50 - Loss: -0.493   lengthscale: 5.243   noise: 0.018
Iter 49/50 - Loss: -0.549   lengthscale: 5.374   noise: 0.016
Iter 50/50 - Loss: -0.608   lengthscale: 5.503   noise: 0.014`

**Comment by jacobrgardner** [comment]
The MLL is the log of a probability density function, not a probability mass function. It can be negative. Consider the PDF of a 0 mean Gaussian evaluated at 0 when \sigma<1/sqrt(2*pi).
